website/public/schemas/yaml/Address.yaml

```diff
@@ -35,6 +35,8 @@ required:
   - stateOrProvince
   - country
   - postalCode
+unevaluatedProperties:
+  not: {}
 examples:
```

Comment:
This explicitly disallows additional properties, which helps us catch mapping errors when extra fields are added (rather than silently allowing them)